TDD/TDD1_BerlinClock/src/test/java/logic/impl/BerlinClockTest.java

```diff
@@ -0,0 +1,156 @@
+package logic.impl;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class BerlinClockTest {
```

Comment:
Tests are ok, but as you change structure of your application, you should refactor those tests.